Zappy_IA_Report/DTC/Code/parser.py

```
@@ -0,0 +1,41 @@
+
+        if not isinstance(machine, str) or machine.strip() == "":
+            exit_error("Error: Invalid machine hostname (-h).")
+        parser = iaParser(
+            port=port,
+            name=name,
+            machine=machine
+        )
+        return parser
+    except ValueError as e:
+        exit_error(f"Error parsing: {e}")
+        sys.exit(84)
```

Flake8 (7.2.0)

[warning] no newline at end of file

(W292)